src/main/java/seedu/planner/model/ModulePlanner.java

```diff
@@ -76,13 +87,12 @@ public void deleteModulesFromSemester(List<Module> modules, int index) {
      * Returns all {@code Module}s taken in the {@code Semester} wrapped in an
      * {@code ObservableList}.
      *
-     * @param index The nominal {@code Semester} index the {@code Module}s
-     *                      are stored at
-     * @return An {@code ObservableList} containing all the {@code Module}s
+     * @param index A valid index.
+     * @return A list of modules taken in the semester.
      */
     @Override
     public ObservableList<Module> getModulesTakenFromSemester(int index) {
```

Comment:
I think can remove the "FromSemester" here as well

Reply:
Removed "FromSemester", changed "get" to "list" to be more descriptive.